modules/virt-virtctl-commands.adoc

```diff
@@ -66,4 +66,13 @@ the specified port of the node.
 
 |`virtctl help`
 |Display a descriptive list of `virtctl` commands.
+
+|`virtctl fslist <vmi_name>`
+|Return full list of filesystems available on the guest machine.
+
+|`virtctl guestosinfo <vmi_name>`
+|Return guest agent info about operating system.
+
+|`virtctl userlist <vmi_name>`
+|Return full list of logged in users on the guest machine.
```

Comment:
logged in users -> logged-in users

Same about "the" or "a" in "Return full list"

Reply:
Changes have been made as suggested. Thanks very much @bergerhoffer. Preview can be viewed at the same link above.

Reply:
> logged in users -> logged-in users

This still needs to be addressed.